ssm_jax/hmm/core.py

```diff
@@ -0,0 +1,226 @@
+import jax.numpy as jnp
+import jax.random as jr
+from jax import lax
+
+
+def hmm_filter(initial_distribution,
+               transition_matrix,
+               log_likelihoods):
+    """_summary_
```

Comment:
Are these docstrings auto-generated by your IDE? (What IDE do you use?)

Reply:
Yes, I'm using VSCode with the autoDocstring plugin.